tests/specs/sass-spec-exclude.txt

```diff
@@ -947,8 +1021,6 @@ libsass/debug-directive-nested/mixin
 libsass/delayed
 libsass/div
 libsass/env
-libsass/features/extend-selector-pseudoclass
-libsass/features/global-variable-shadowing
```

Comment:
These removals are not about increased compliance on our side. It is just that I identified the `libsass/features/*` specs as being duplicates of `core_functions/meta/feature-exists/*` ones and I removed them upstream.